.github/ISSUE_TEMPLATE/feature_template.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Do we need this? I think we use only `story_template` to generate our feature task. 🤔

Reply:
This is all from the template with nothing removed 😅 Shall we remove this in #6? 😄

Reply:
@ryan-conway Sure thing. 😬